data/json/item_actions.json

```diff
@@ -5,6 +5,10 @@
         "name" : "Sew"
     },{
         "type" : "item_action",
+        "id" : "SEW_ADVANCED",
+        "name" : "Sew_ADVANCED"
```

Comment:
name is shown to the player, and translated.  Something like "Modify Clothing" would be better.

(The id field is internal; though it's good practice to have id and name match as closely as feasible, SEW_ADVANCED is OK there.)